apps/meteor/server/startup/migrations/v283.ts

```diff
@@ -0,0 +1,18 @@
+import { Settings, Permissions } from '@rocket.chat/models';
+
+import { addMigration } from '../../lib/migrations';
+
+addMigration({
+	version: 283,
+	async up() {
+		// Removing all settings & permissions related to Legacy FB Messenger integration
+		await Promise.all([
+			Settings.deleteMany({
+				_id: {
+					$in: ['Livechat_Facebook_Enabled', 'Livechat_Facebook_API_Key', 'Livechat_Facebook_API_Secret'],
+				},
+			}),
+			Permissions.removeById('view-livechat-facebook'),
```

Comment:
Should we close conversations opened by this integration? Otherwise they could be dangling forever as no new messages will be received/sent 🤔

Reply:
That's a good point. Updated the PR to close any open rooms connected to old integration :heavy_check_mark: